app/views/shared/provider/_user_widget.html.slim

```diff
@@ -43,33 +52,21 @@ div id="user_widget"
 
         = fixed_width_icon_link_to 'Sign Out', 'times', provider_logout_path, class: 'PopNavigation-link', title: "Logout #{current_user.username}"
 
-  ul id="top-menu-nav"
-    li class=('active' if active_menu?(:topmenu, :dashboard) )
-      = fixed_width_icon_link_to 'Dashboard', 'home', provider_admin_dashboard_path
-
   // HACK of the month. See features/support/current_user.rb if you are hungry for reasons
   div class="username" style="display:none;"
     = current_user.username
 
-
 javascript:
+  document.addEventListener('DOMContentLoaded', function () {
+    ApiSelector({currentApi: #{json @service}, apis: #{json @all_services}, controllerName: #{json controller_name}}, 'api_selector')
+  })
+
   $(document)
-    .on('click', '.u-toggler', function(event) {
-      $(this).toggleClass('is-toggled');
-      var toggleable = $(this).siblings('.u-toggleable')
-      toggleable.toggleClass('is-toggled');
-      toggleable.find('input').filter(':visible:first').focus();
-      return false;
+    .on('click', '.u-toggler', function (e) {
+      e.stopPropagation()
+      toggleNavigation(e.currentTarget)
```

Comment:
`toggleNavigation` is undefined, I'm trying to fix it.

Reply:
It's because the wiring up of webpack and eslint properties. Feel free to do it :)